network/datadog_checks/network/network.py

```diff
@@ -49,6 +49,11 @@ def check(self, instance):
             instance = {}
 
         self._excluded_ifaces = instance.get('excluded_interfaces', [])
+        if not isinstance(self._excluded_ifaces, list):
```

Comment:
Shouldn't this happen at __init__ time?

Reply:
Funny… Looks like the `Network` check is pretty old school as it actually uses the `instance` passed to `.check()` instead of `self.instance`… Looks like a candidate for a refactor IMO, though this PR is consistent with the rest of the config handling done here so far.

Reply:
Yeah we can do that later.